packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -160,7 +168,18 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
         );
       });
     });
-  }, [remoteParticipants, onRenderRemoteVideoTile, onCreateRemoteStreamView, remoteVideoViewOption]);
+  }, [remoteParticipants, onRenderRemoteVideoTile, onCreateRemoteStreamView, remoteVideoViewOption, onRenderAvatar]);
+
+  if (layout === 'floatingLocalVideo' && remoteParticipants && remoteParticipants.length > 0) {
+    return (
+      <>
+        <Stack style={{ position: 'absolute', bottom: 0, right: 0 }}>
```

Comment:
Will a new styles object be created each time and cause spurious renders here?

Reply:
Not for this. It applies to Component props that are being watched.

Reply:
But I will move to a class nonetheless.